diff --git a/trunk/user/www/n56u_ribbon_fixed/Advanced_WSecurity_Content.asp b/trunk/user/www/n56u_ribbon_fixed/Advanced_WSecurity_Content.asp
index 21fd630554..918d5bb316 100644
--- a/trunk/user/www/n56u_ribbon_fixed/Advanced_WSecurity_Content.asp
+++ b/trunk/user/www/n56u_ribbon_fixed/Advanced_WSecurity_Content.asp
@@ -133,10 +133,7 @@ function done_validating(action){
 
                                     <table class="table">
                                         <tr>
-                                            <td style="margin-top: 10px; border-top: 0 none;">
-                                                <input class="btn btn-info" type="button"  value="<#GO_2G#>" onclick="location.href='Advanced_WSecurity2g_Content.asp';"/>
-                                            </td>
-                                            <td style="border-top: 0 none;">
+                                            <td style="border-top: 0 none; text-align: center;">
                                                 <input class="btn btn-primary" style="width: 219px" type="button" value="<#CTL_apply#>" onclick="applyRule()" />
                                             </td>
                                         </tr>
